pylsp_black/plugin.py

```diff
@@ -64,29 +64,33 @@ def pylsp_settings():
 
 
 def format_document(client_config, document, range=None):
+    text = document.source
+    config = load_config(document.path, client_config)
+    lines = [(range["start"]["line"] + 1, range["end"]["line"])] if range else ()
```

Comment:
Black says "indices are 1-based and inclusive on both ends" (https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#line-ranges), `range` uses a 0-based index.

Reply:
Also black says `lines` is a list of tuple but uses a tuple as default argument in https://github.com/psf/black/pull/4020/files.

Reply:
> Black says "indices are 1-based and inclusive on both ends" (https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#line-ranges), `range` uses a 0-based index.

Can you add this as a comment in the code?

Reply:
Just added.